### ISSUE TITLE
Warning in BarSeries & DensitySeries when displaying object arrays

### ISSUE THREAD

**Issue by runako** [williaster/data-ui#156]
Thanks for your hard work on this component! It looks great and has proven super useful for my project.

I'm displaying a histogram using a BarSeries and a DensitySeries, based on the example:

https://williaster.github.io/data-ui/?selectedKind=histogram&selectedStory=normal&full=0&addons=0&stories=1&panelRight=0

The data I pass to the rawData prop of each component is an array of objects, and I'm using the valueAccessor property on the histogram component to specify how to get the data for rendering. This works great, but it causes warnings to be generated in the the console log:

Warning: Failed prop type: Invalid prop `rawData[0]` supplied to `DensitySeries`.

In summary, the functionality works great but the warnings in the console are suboptimal.

**Comment by williaster** [comment]
Thanks for checking out `@data-ui` @runako and raising this issue 🐛 I should be able to fix this and get a release out in the next day or two. Will ping you here when it's out 👍
